### PR TITLE
remove assertion that upgrades are fast

### PR DESCRIPTION
**Pull request by mtbc**
see http://lists.openmicroscopy.org.uk/pipermail/ome-users/2016-March/005870.html

### CODE REVIEW

**Comment by hflynn** [comment]
Do we want to go as far as making clear that they may take some hours?

**Comment by mtbc** [comment]
Good point, quite possibly; I'll let this sit for a little while as others may chime in!

**Comment by sbesson** [comment]
I assume we have/can generate our own measurements - similar to what we have done in https://www.openmicroscopy.org/site/support/omero5.2/sysadmins/search.html#monitoring-re-indexing? /cc @kennethgillen

**Comment by mtbc** [comment]
We _could_ but I suspect there may be significant variation among production databases, not just in overall size but also qualitatively. (For instance, big images or many small planes? Lots of metadata or little? Extensive use of annotations?) Also keeping figures up to date adds to our ongoing costs. So, not saying "don't", just pointing out it may be more trouble than it's worth.

Before release it's certainly worth running scripts against nightshade-clone DBs though to pinpoint the slowest statements for nightshade in case a second look at specifically those helps us to see an easy win that would benefit many.

**Comment by sbesson** [comment]
Digging through the logs of our [CI job testing the upgrade scripts](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-upgrade/), the upgrade of nightshade was taking typically a few hours

```
2015-11-13 06:14:29,133 [    omego.db] INFO  Current omero db version: ('OMERO5.0', '0')
2015-11-13 06:14:29,137 [    omego.db] INFO  Upgrading database using /opt/hudson/workspace/OMERO-DEV-merge-upgrade/OMERO.server/sql/psql/OMERO5.1__1/OMERO5.0__0.sql
2015-11-13 06:14:29,137 [omego.extern] INFO  Executing [custom environment]: psql -d OMERO-DEV-merge-upgrade -h localhost -U omero -w -A -t -f /opt/hudson/workspace/OMERO-DEV-merge-upgrade/OMERO.server/sql/psql/OMERO5.1__1/OMERO5.0__0.sql
2015-11-13 09:07:51,470 [    omego.db] INFO  Upgrading database using /opt/hudson/workspace/OMERO-DEV-merge-upgrade/OMERO.server/sql/psql/OMERO5.2__0/OMERO5.1__1.sql
```

As discussed with @mtbc, precising this order of magnitude for major upgrades might be sensible.

File: omero/sysadmins/server-upgrade.txt [hunk 1]
```diff
@@ -218,8 +218,8 @@ Run the upgrade script
 """"""""""""""""""""""
 
 You **must** use the same username and password you have defined during
-:doc:`unix/server-installation`. The |version| upgrade script should execute
-in a short time.
+:doc:`unix/server-installation`. For a large production system you
+should plan for the upgrade script to take hours to run.
```
Comment (manics) [inline omero/sysadmins/server-upgrade.txt]: Sounds a bit too negative, how about "may take several hours to run"?

Comment (hflynn) [inline omero/sysadmins/server-upgrade.txt]: That wording would work for me - "For a large production system you should plan for the fact that the upgrade script may take several hours to run." ?

**Comment by hflynn** [comment]
:+1: Just rebuilding staging before merging.

**Pull request merged**